src/daft-logical-plan/src/optimization/rules/granular_projections.rs

```diff
@@ -86,7 +87,7 @@ impl SplitGranularProjection {
                                 changed = true;
                                 // Child may not have an alias, so we need to generate a new one
                                 // TODO: Remove with ordinals
-                                let child_name = child.semantic_id(projected_schema).id;
+                                let child_name = format!("id-{}", uuid::Uuid::new_v4());
```

Comment:
@kevinzwang I ended up undoing this because the semantic ID for url_downloads is very long and hard to read. It makes the plans worse to parse through.